src/views/Auth.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { signInUser, signUpUser } from '../services/userRoute';
+import AuthForm from '../components/AuthForm';
+import classNames from 'classnames';
+import './Auth.css';
+
+export default function Auth({ setCurrentUser }) {
+  const [type, setType] = useState('signin');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      let resp;
+      if (type === 'signin') {
+        resp = await signInUser(email, password);
+      } else {
+        resp = await signUpUser(email, password);
+      }
+      setCurrentUser(resp);
```

Comment:
So there's a tiny bug that doesn't render the page when initially logging in. It's because the shape of the data we get from `getUser()` is slightly different from the response we get back from `signInUser`. We can resolve this by setting the current user to `{user: resp}` instead of just `resp`